src/main/java/dev/warrant/model/object/WarrantListResult.java

```diff
@@ -0,0 +1,47 @@
+package dev.warrant.model.object;
+
+import dev.warrant.model.Warrant;
+
+public class WarrantListResult {
```

Comment:
I think we can get rid of this class and instead use `ListResult<Warrant>` once we update `ListResult` to take any type (and not just those that implement `WarrantObject`).

Reply:
We have `WarrantListResult` because this gets passed into `makeGetRequest` to map the response into the specified class, which we wouldn't be able to do with `ListResult<Warrant>`